packages/InteractiveProfilingTool-Core.package/IPTMessageTallyWrapper.class/instance/getMemoryStats.st

```diff
@@ -0,0 +1,20 @@
+information extraction
+getMemoryStats
+
+	| memoryEnd oldSpaceEnd youngSpaceEnd |
+	
+	self messageTally gcStats ifNil: [^ nil].
```

Comment:
For consistency reasons please change to
```smalltalk
self messageTally gcStats 
    ifNil: [^ nil].
```

Reply:
Ok ist getan.